app/Metadata/Extractor.php

```diff
@@ -341,7 +340,7 @@ public static function createFromFile(NativeLocalFile $file): self
 							// that timezone.
 							$taken_at->setTimezone(new \DateTimeZone(date_default_timezone_get()));
 						}
-					// In the remaining cases the timezone information was
+						// In the remaining cases the timezone information was
```

Comment:
Spurious change?

Reply:
Don't ask me why. Make formatting is forcing it. :(

Reply:
I literally tried to revert it.